### PR TITLE
Add support for symfony 5

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes -
| Related issues/PRs | -
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Update recipes to newest versions, so new sulu installations get configuration from symfony 5.

#### Why?

Add support for symfony 5.

#### To Do

- [ ] Create a documentation PR
- [ ] Create PR for sulu/sulu test skeleton

### CODE REVIEW

File: config/bootstrap.php [hunk 1]
```diff
@@ -1,22 +0,0 @@
-<?php
```
Comment (alexander-schranz) [inline config/bootstrap.php:1]: The bootstrap file was [removed](https://github.com/symfony/symfony-docs/issues/13759) and replaced in symfony 5.1 by a new DotEnv->bootEnv method.

File: composer.json [hunk 2]
```diff
@@ -29,30 +29,30 @@
         "php": "^7.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "dantleech/phpcr-migrations-bundle": "^1.0",
-        "doctrine/doctrine-bundle": "^1.10 || ^2.0",
+        "dantleech/phpcr-migrations-bundle": "^1.2",
+        "doctrine/doctrine-bundle": "^2.0",
```
Comment (alexander-schranz) [inline composer.json:33]: As discussed with @chirimoya as the recipes only match the new version we will require the newest symfony package versions in the skeleton for new projects.

Comment (danrot) [reply composer.json:33]: This is not really a BC break, because existing projects don't necessarily have to update, right?

Comment (alexander-schranz) [reply composer.json:33]: Yes this is not a BC Break, the skeleton should from my side always represent how a new project should look like.

**Comment by alexander-schranz** [comment]
Currently the CI is failing because of:

https://github.com/symfony/symfony/issues/37166

We could currently work around by inlining the event name and not using the parameter: 

https://github.com/massiveart/MassiveSearchBundle/pull/146

**Pull request merged**